test/box/fix-not-deleting-unix-socket-path.lua

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env tarantool
+
+require('console').listen(os.getenv('ADMIN'))
+
+box.cfg({
+    listen = os.getenv('LISTEN'),
+    iproto_threads = tonumber(arg[1]),
+})
+
+box.schema.user.grant('guest', 'read,write,execute,create,drop', 'universe')
+
+box.ctl.on_shutdown(function()
+	box.schema.user.revoke('guest', 'read,write,execute,create,drop', 'universe')
```

Comment:
Why do you need this on shutdown? The newly created server is deleted in the end of the test anyway, with all its snaps and xlogs and therefore users and rights.

Reply:
If you grant some rights and then restart server. It's fails because of error message ` Start failed: User 'guest' already has read,write,execute,create,drop access on universe`. So you need always revoke it when it's finish work!

Reply:
Nope, there is `if_not_exists` option so as not to get this error. But the better question - why do you need the access rights anyway? You don't use replication, and don't call functions remotely (test-run's `eval` uses admin console).

Reply:
You right, i don't need it! Also i don`t need net_box in test. I think about add some net_box connections to check your test case, after fix:
```lua
box.cfg{listen = 3313}
box.cfg{listen = 'baduribaduri:1'}
require('net.box').connect(3313)
```
but i think it's better to do in separate commit and separate test after discussion with you.